packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -402,6 +402,7 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
             url,
           }))
 
+        // TODO: isn't it equivalent to just `inputDatasources` if the first argument is `[]`?
```

Comment:
I agree!

Reply:
Could be a good candidate for a deep dive